### PR TITLE
Views

### PR DESCRIPTION
**Pull request by assadounto**
In this feature, I added views

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Rich,
 
 While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
 
 ## Invalid Code Review Request
You have submitted a project, whereby Stylelint linter checks are not showing. Kindly ensure that github actions are properly configured and that **all** required linters are showing and that checks are passing Kindly request for another code review when this fix is done.

 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _Invalid Code Review Request **does not count** into the code reviews limit._

File: app/views/users/_user.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+        <div class="img-cont">
+           <img class="image" src="<%= user.photo %>" alt="Profile picture">
+        </div>
+
+        <div class="info-cont">
+           <div><%= user.name %></div>
+           <div> <%= pluralize(user.postCounter, 'Post') %></div>
+        <div>
+  <% end %>
+</div>
```
Comment (ahmetbozaci) [inline app/views/users/_user.html.erb:12]: - [ ] Kindly make sure users' page match the wireframe

image

image

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<%= render @user %>
+<%= render @user.posts%>
+ <%=link_to 'see all posts' ,user_posts_path(@user)%>
```
Comment (ahmetbozaci) [inline app/views/users/show.html.erb:3]: - [ ] Kindly make sure user show page match the wireframe. 
- [ ] Kindly do not forget to put the see all posts button.

image

image

File: app/views/users/_user.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<div class="d">
+  <%= link_to "/users/#{user.id}" do %>
+        <div class="img-cont">
+           <img class="image" src="<%= user.photo %>" alt="Profile picture">
+        </div>
+
+        <div class="info-cont">
+           <div><%= user.name %></div>
+           <div> <%= pluralize(user.postCounter, 'Post') %></div>
```
Comment (ahmetbozaci) [inline app/views/users/_user.html.erb:9]: - [ ] The post counter method is not working you can see on the page it is still showing zero. Kindly fix it. OPTIONAL

image

File: app/views/posts/_post.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+   comments: <%=post.comments_counter %>
+    likes: <%=post.likes_counter%>
+ <%post.comments.each do |comment|%>
+   <div>
+   <%=comment.user.name%>
+    <%=comment.text%>
+   </div> 
+    <%end%>
+<% end %>
+
```
Comment (ahmetbozaci) [inline app/views/posts/_post.html.erb:13]: - [ ] Kindly make sure post index page match the wireframe
image
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @assadounto  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
✔️ Your git-flow is correct. 
✔️ You set up linter files and no errors. 
✔️ Your pull request is very professional. It has a meaningful title and summary. Keep it up.
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] Everything is working well. Kindly follow the wireframe. You can skip the style if you want.

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
@assadounto 
-  [ ] Kindly add more detail to the pull request description. 
⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ 

- Your commit messages should always start with capital letters and they should be an imperative tense. I see some of them are not like that. Instead of writing `Added style or Created the file` kindly write `Add style or Create the file`
- ***I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this in your next projects.***
- Kindly change them by using `git-rebase`. If you want to know how to change ı leave a few resources.
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)

⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ 
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @assadounto, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**